release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+
+push-multiarch-%:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
This doesn't do anything, the variable is getting set and the shell in which it was set exits. You need to join the lines with `; \`:
```
push-multiarch-%:
    gcloud auth configure-docker
    set -ex; \
    DOCKER_CLI_EXPERIMENTAL=enabled; \
    export DOCKER_CLI_EXPERIMENTAL; \
    docker buildx create --use --name multiarchimage-buildertest; \
    pushMultiArch () { \
```

I intentionally avoided `export DOCKER_CLI_EXPERIMENTAL=enabled` because that only works in bash.